### ISSUE TITLE
NF-core tools fails to parse bowtie2/align/main.nf when trying to download images

### ISSUE THREAD

**Issue by skilpinen** [nf-core/cutandrun#181]
### Description of the bug

https://github.com/nf-core/cutandrun/blob/master/modules/nf-core/bowtie2/align/main.nf

### Command used and terminal output

```console
nf-core download cutandrun -r 3.0 --outdir ~/nf-core-pipelines/

ERROR    Cannot parse container string in '/home/kilpinen/nf-core-pipelines/workflow/modules/nf-core/bowtie2/align/main.nf':

             ${ workflow.containerEngine ==

         ⚠ Skipping this singularity image..
INFO     Found 19 containers
Pulling singularity images ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╺━━━━━  95% • 18/19 completed
quay.io/biocontainers/python:3.8.3 Copying from cache to target directory ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
CRITICAL [Errno 2] No such file or directory: '/home/kilpinen/singularity_images/quay.io-biocontainers-python-3.8.3.img'
```

**Comment by chris-cheshire** [comment]
Can you provide anymore context with this? I cant really do much with what you have posted...

**Comment by skilpinen** [comment]
Hi,

Yes, sorry. So I have just installed nf-core tools with pip (in Ubuntu Linux 20.04). I followed instructions of cutandrun pipeline with regards to Singularity containers that those should be predownloaded with nf-core tools.  

So I run following:
nf-core download cutandrun -r 3.0 --outdir ~/nf-core-pipelines/

And it completes otherwise but fails on one singularity image:
ERROR    Cannot parse container string in '/home/kilpinen/nf-core-pipelines/workflow/modules/nf-core/bowtie2/align/main.nf'

I looked that file, which it fetches directly from this cutandrun repo, and only difference in structure of the container string within the file is that it has like double quotes inside of each other (" "" "), whereas in other .nf files it is like " '' ". Whether this is the problem, I don't know, but I know that nf.core tools fails to download cutandrun pipeline because of this one error.
